ui/src/infra-model/translations.fi.json

```diff
@@ -156,6 +158,10 @@
                 "no-structure-alignment": "Vaihteen {{switchName}} pääosat ({{jointNumbers}}) raiteella {{alignmentName}} eivät vastaa mitään vaihdetyypin linjaa",
                 "alignment-joint-mismatch": "Vaihteen {{switchName}} pääosat ({{jointNumber}}) raiteella {{alignmentName}} ovat eri sijainnissa kuin muilla raiteilla",
                 "alignment-joint-inaccurate": "Vaihteen {{switchName}} pääosat ({{jointNumber}}) raiteella {{alignmentName}} ovat sijainniltaan epätarkat suhteessa muihin raiteisiin"
+            },
+            "km-post": {
```

Comment:
Näiltä kaikilta itse asiassa puuttui käännösavaimet. Lopuille käännösavaimia ei lisätty vielä täällä koska en tiedä miten ne kuuluisi kääntää. Teen niiden lisäämisestä tiketin

Reply:
Oliko nää nykyään kilometripylväitä vai tasakilometripisteitä?

Reply:
tasakilometripiste olisi oikea